swift/README.md

```diff
@@ -0,0 +1,9 @@
+## Warning
+
+The Swift codeql package is an experimental and unsupported work in progress.
```

Comment:
```suggestion
The Swift codeql package is experimental and unsupported work in progress.
```

Reply:
hmm, "work in progress" is a substantive, not an adjective (although we do tend to use it as if it were one), so to be really nitpickingly correct I think the indeterminate article should be there...

Reply:
I certainly don't feel hard about this one, feel free to discard this :)